### ISSUE TITLE
[1.20.1] Cannot disable adventure module

### ISSUE THREAD

**Issue by IchHabeHunger54** [Shadows-of-Fire/Apotheosis#989]
It appears that a variant of #955 has resurfaced in 1.20.1: When disabling the Adventure module, datapack validation fails. Apotheosis version is 7.0.4 (latest at the time of writing), Forge version is 47.1.3.

From the logs (I removed the stacktraces because they're all pretty much the same and don't really contribute anything, worldgen stacktraces being what they are):
```
> Errors in registry minecraft:root:
>> Errors in element minecraft:worldgen/configured_feature:
java.lang.IllegalStateException: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/configured_feature]: [apotheosis:boss_dungeon, apotheosis:boss_dungeon_2, apotheosis:rogue_spawner]
>> Errors in element minecraft:worldgen/processor_list:
java.lang.IllegalStateException: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/processor_list]: [apotheosis:item_frame_gems]
>> Errors in element minecraft:worldgen/structure:
java.lang.IllegalStateException: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/structure]: [apotheosis:tower_leaf, apotheosis:tower_main, apotheosis:tower_sand, apotheosis:tower_spruce]
>> Errors in element minecraft:worldgen/template_pool:
java.lang.IllegalStateException: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/template_pool]: [apotheosis:tower_leaf, apotheosis:tower_main, apotheosis:tower_sand, apotheosis:tower_spruce]
> Errors in registry minecraft:worldgen/configured_feature:
>> Errors in element apotheosis:boss_dungeon:
java.lang.IllegalStateException: Failed to parse apotheosis:worldgen/configured_feature/boss_dungeon.json from pack Apotheosis-1.20.1-7.0.4.jar
Caused by: java.lang.RuntimeException: Unknown registry key in ResourceKey[minecraft:root / minecraft:worldgen/feature]: apotheosis:boss_dungeon
>> Errors in element apotheosis:boss_dungeon_2:
java.lang.IllegalStateException: Failed to parse apotheosis:worldgen/configured_feature/boss_dungeon_2.json from pack Apotheosis-1.20.1-7.0.4.jar
Caused by: java.lang.RuntimeException: Unknown registry key in ResourceKey[minecraft:root / minecraft:worldgen/feature]: apotheosis:boss_dungeon_2
>> Errors in element apotheosis:rogue_spawner:
java.lang.IllegalStateException: Failed to parse apotheosis:worldgen/configured_feature/rogue_spawner.json from pack Apotheosis-1.20.1-7.0.4.jar
Caused by: java.lang.RuntimeException: Unknown registry key in ResourceKey[minecraft:root / minecraft:worldgen/feature]: apotheosis:rogue_spawner
> Errors in registry minecraft:worldgen/processor_list:
>> Errors in element apotheosis:item_frame_gems:
java.lang.IllegalStateException: Failed to parse apotheosis:worldgen/processor_list/item_frame_gems.json from pack Apotheosis-1.20.1-7.0.4.jar
Caused by: java.lang.RuntimeException: Not a json array: {"processors":[{"processor_type":"apotheosis:item_frame_gems"}]}
```

**Comment by Shadows-of-Fire** [comment]
Root caused as a forge issue: https://github.com/neoforged/NeoForge/issues/114

**Comment by Camawama** [comment]
So do you suggest only using NeoForge?

**Comment by SiverDX** [comment]
if you want to disable the module the definitely 

in general id recommend neoforge since this mod is based off of that 
it's not guaranteed for all features to correctly function when using forge

**Comment by Camawama** [comment]
I just can stand the split in the modding community...it's ruining modding. I can't switch to NeoForge. Is there any possible work around for this? How difficult would it be for someone to create a fork that fixes the issues?

**Comment by Camawama** [comment]
Forge Forgotten Forever.

**Comment by Camawama** [comment]
very annoying

**Comment by lockf** [comment]
It's fixed in Forge  47.2.23

**Comment by Camawama** [comment]
> It's fixed in Forge 47.2.23

Thank you for the heads up. I'll do some testing later.
